.mergify.yml

```diff
@@ -1,3 +1,13 @@
+shared:
+  - commit_message_template: &commit_message_template |
+      {{title}}
+      
+      {{body | get_section("## What's changed and what's your intention?")}}
+      
+      {% for user in approved_reviews_by %}
+      Approved-By: {{user}}
+      {% endfor %}
+
```

Comment:
```suggestion

{% for commit in commits %}
Co-Authored-By: {{ commit.author }} <{{ commit.email_author }}>
{% endfor %}
```

🤔

(But maybe also need to dedup (and exclude bot?)...

Reply:
I guess you need [unique filter](https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.unique)?

Reply:
Won't this result in 

```
Co-Authored-By: a a.b@mail.com
Co-Authored-By: b c.d@mail.com
Co-Authored-By: c d.e@mail.com
```

Don't we want 

```
Co-Authored-By: 
{% for commit in commits %}
{{ commit.author }} <{{ commit.email_author }}>
{% endfor %}
```

Reply:
<img width="663" alt="image" src="https://user-images.githubusercontent.com/37948597/207024736-c2462930-63b9-45c4-a0bf-b577a03e1779.png">

Reply:
Okay. If that is the layout we want then this is good 👍

Reply:
Updated. Please take a look